CMake/SlicerCPack.cmake

```diff
@@ -77,6 +77,13 @@ if(Slicer_BUILD_QT_DESIGNER_PLUGINS)
   file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/${Slicer_BIN_DIR})
   # Configure designer launcher
   find_package(CTKAppLauncher REQUIRED)
+  # At the time this script is executed for both Slicer's and SlicerCAT's binary dir 
+  # is set to './d/Slicer-build' and there application settings resides
+  set(app_name "Slicer")
+  if(DEFINED Slicer_MAIN_PROJECT_APPLICATION_NAME AND 
+    NOT Slicer_MAIN_PROJECT_APPLICATION_NAME STREQUAL ${app_name})
```

Comment:
Is this line needed? Does it have a side effect if we set the variable to the same value?

Also note that `NOT Slicer_MAIN_PROJECT_APPLICATION_NAME STREQUAL ${app_name}` is always true because `Slicer_MAIN_PROJECT_APPLICATION_NAME` is just a string. To get the variable value `${Slicer_MAIN_PROJECT_APPLICATION_NAME}` should have been used.